### PR TITLE
feat: CI

### LINKED ISSUE #11: Setup CI
It would be awesome with some CI when we push branches. It's up to the implementer to define what we need. It could maybe be some prettier formatting to make sure stuff is formatted or maybe a ts check. Something like that. 
Explore Astro docs and template repos.

**Accepttest**
* Add a CI that fits the project.

### PR DESCRIPTION
**Pull request by kristianbinau**
## Description
Added CI

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [x] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: src/components/UserSettings.astro [hunk 1]
```diff
@@ -2,17 +2,17 @@
 import { url } from '@lib/helpers';
```
Comment (frederikpyt) [inline src/components/UserSettings.astro:2]: Very big file 💀

File: src/pages/u/index.astro [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 import LayoutSidebar from '@layouts/LayoutSidebar.astro';
 import Dashboard from '@components/Dashboard.astro';
+// @ts-ignore https://github.com/withastro/language-tools/issues/476
```
Comment (frederikpyt) [inline src/pages/u/index.astro]: 🤢

**Review by frederikpyt — APPROVED** [review]

File: src/pages/u/index.astro (hunk 2 above)
Comment (kristianbinau) [reply src/pages/u/index.astro]: Nothing we can do : /

File: src/components/UserSettings.astro (hunk 1 above)
Comment (kristianbinau) [reply src/components/UserSettings.astro:2]: It's good : )

**Pull request merged**